content/docs/introduction/central.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Open mSupply Central Server"
+description = "Open mSupply central server functionality"
+date = 2022-06-10T11:38:00+00:00
+updated = 2022-06-10T11:38:00+00:00
+draft = false
+weight = 8
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = true
++++
+
+Open mSupply central server is a special site that allows for configurations of subset of central data. Please see [requirements](/docs/introduction/requirements/#open-msupply-requirements) section for extra info.
+
+## What is open mSupply central server
+
+In essence it's just another open mSupply instance. What makes it difference from a typical remote site ?
+
+- Only one instance of open mSupply central server will exist in open mSupply system
+- It will be configured by our support staff and needs to be available on a world wide web (typically as a cloud server, but can also be hosted in country)
+- All remote instance of open mSupply will communicate with open mSupply central server, as part of the synchronisation process
+- Allows for configurations of subset of central data
+
+## How does open mSupply central server look
+
+It will look very much like any other open mSupply instance, you will see distinct coloured bar at the bottom of interface which identifies the site as central server
+
+![central server identifier bar!](/docs/introduction/images/central_server_bar_identifier.png)
+
+Similar to mSupply central server, only one instance of open mSupply central server will exist in open mSupply system. 
+Our support staff will be control 
```

Comment:
```suggestion
Similar to the mSupply central server, only one instance of the Open mSupply central server will exist in the Open mSupply system. Contact our support staff for available customisations.
```

Reply:
Hmm, not 100% sure about this one, i'll remove `Our support staff will be control`, which is unfinished. I am not sure if `Contact our support staff for available customisations.` is quite accurate. I wanted to say that users don't need to be concerned with installing the central server site

5816ade7c1a7f52b3494a826c8c586f87980b036

Reply:
Ah yeah I wasn't sure where you were going with that XD